src/mediaplayer.cs

```diff
@@ -1578,6 +1578,7 @@ interface MPPlayableContentDelegate {
 		void InitializePlaybackQueue (MPPlayableContentManager contentManager, Action<NSError> completionHandler);
 
 		[iOS (9,3)]
+		[Deprecated (PlatformName.iOS, 12, 0, message: "Use Intents framework API instead.")]
```

Comment:
Minor: would probably have added a "the": "Use the Intents framework APIs instead."